### DISCUSSION TITLE
Transmission refusing to download some torrents.

### DISCUSSION THREAD

**Discussion by dantearmok** [transmission/transmission#5146] [Support]
2.5-GHz Core i5 Mac mini (2012), 16GB RAM, plus an assortment of USB & FW HDs.
macOS Catalina, Safari, ClamXAV, and PIA (all fully updated).

Running Transmission 4.0.1-dev (7f9ef4730e) -- March 5th's nightly.

I regularly surf several torrent sites, and have two HDs "dedicated" to holding downloaded torrent materials, ~3 to 4 weeks of storage.  ~250 seeding and ~35 leeching.  

All of the torrents I've added recently start downloading fairly quickly *except* those from rarbg.  Those don't download at all, and the ones from that site that should be seeding are doing nothing.

To narrow this problem down, I installed Deluge.  It has no problem with rarbg torrents.

Any ideas as to what could be causing this?  Is there more information I can provide?

** EDIT** Mar 20, 8:40 ET

Running Transmission-c60bb5b834 Build #8952 (Mar 19, 2023, 3:37:21 PM)

Folks, today I'm having the same nil download/upload issue, of both rarbg and ubuntu items (only those; torrents from other sites run just fine).  I see that in the Trackers subwindow that all the entries for the rarbg items say "N/A" for seeders.  I'm guessing that would be indicative of a tracker problem (outage, ddos, etc)?  But the seeders on the ubuntu items look healthy, 1978 and 444 respectively.  So maybe this isn't a tracker issue?

** EDIT ** Mar 25, 11pm ET

Found this: https://github.com/transmission/transmission/discussions/4916

Tonight, neither rarbg nor ubuntu items are leeching.

Any ideas as to what could be causing this?

Thanks,
- Dan.

**Comment by Pentaphon** [comment]
The Ubuntu torrent is very odd, even on other clients. It only downloads from 1 peer for what seems to be forever until it finally starts connecting to more peers. I think that might be a tracker issue for that one. However, I have no trouble downloading rarbg stuff from Windows or Linux.

**Comment by dantearmok** [comment]
Yeah, I see that about the Ubuntu torrents.  

In qBittorrent, they're slow to catch then build.  The Rarbg torrents take off fast, connecting to peers within a few seconds of starting.  The download speed is pretty good too.

But in Transmission, I gots nothing.  Except for updating to the latest nightly, the app is running 24/7.  It seeds older Rarbg items out of cache and pez. Over the last three days it's downloaded zilch from new rarbg torrents or the ubuntu ones.  Very frustrating.

**Comment by dantearmok** [comment]
Last night (03/27) I punted.  Shutdown both Transmission and qB.  4 day outrage. :(

Tonight ... Transmission is working fine downloading both rarbg and ubuntu items!

Just updated to Transmission-e68c72daa4 Build #8971 (Mar 28, 2023, 6:42:03 PM)

- Dan.

**Reply by dantearmok** [reply]
Transmission is failing to leech/stream Rarbg torrents again.  

This outage started Saturday afternoon and continues (it's now 1am Mon).

Tried 3.0, 4.0.3, and the latest nightly.

qB leeches and streams them just fine.  Currently 4 leeching, 56 seeding, out of 229 torrents.

**Comment by dantearmok** [comment]
![Screen Shot 2023-04-17 at 3 04 22 AM](https://user-images.githubusercontent.com/20881295/232409740-72c8297e-a116-4435-a475-2e97cf5c3556.jpg)

This is what I'm seeing on all the torrents that are failing to operate right now.  In the torrents that are working (from not rarbg sites), their tracker list is (mostly) longer and at least one shows seeders'n'leechers.

And I found this - All trackers get connection failed #3285

**Comment by dantearmok** [comment]
Definitely a tracker issue.  By adding "http://tracker.openbittorrent.com:80/announce" to some of the idle torrents, they recover and start leeching/seeding.
